CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# dbt_hubspot v0.15.0
+
+([PR #127](https://github.com/fivetran/dbt_hubspot/pull/127)) includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+- When including a passthrough `property_hs_*` column, you now have the option to include the corresponding, human-readable label in the staging models. 
+  - See the [Adding property label section](https://github.com/fivetran/dbt_hubspot#adding-property-label) of the README for instructions on how to enable this feature! 
+  - This update applies to models:
+    - `hubspot__company`
+    - `hubspot__contact`
+    - `hubspot__deal`
+    - `hubspot__ticket` 
```

Comment:
Same as source, I don't feel this is really a breaking change and more of a feature.